changelogs/fragments/214-add_firewalld_info_module.yml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - firewalld_info - add ``firewalld_info`` module to ``ansible.posix`` collection (https://github.com/ansible-collections/ansible.posix/issues/98)
```

Comment:
New modules do not require a changelog.

Reply:
@Akasurde , thank you for your kindly advice every time!
I have removed this fragment file, but then the `ansible-changelog-fragment` job is failing in Zuul.
It seems that the root cause of this issue is that missing fragments file in the pull request.  I  would appreciate it if you could give me some advice to avoid this issue.

* https://368fbbd6c602bf2d6e11-6df4799967521a9dda23b4dfc17b8166.ssl.cf2.rackcdn.com/214/16d09a585d82686ae540e99c1a40786d51af4400/check/ansible-changelog-fragment/21d4f61/job-output.txt

```
2021-07-09 12:05:37.896752 | TASK [Changelog fragment failed]
2021-07-09 12:05:37.993756 | localhost | ERROR
2021-07-09 12:05:37.994061 | localhost | {
2021-07-09 12:05:37.994133 | localhost |   "msg": "Your pull-request is missing a changelog fragment, please add one. It should explain to end users the reason for your change."
2021-07-09 12:05:37.994216 | localhost | }
2021-07-09 12:05:38.018566 | 
2021-07-09 12:05:38.018678 | PLAY RECAP
2021-07-09 12:05:38.018883 | localhost | ok: 1 changed: 1 unreachable: 0 failed: 1 skipped: 0 rescued: 0 ignored: 0
```

Reply:
Ok. Let us restore `trivial` changelog and I will raise a feature request against `ansible-changelog-fragment` job to exclude new module.

Reply:
Sure, I have restored it as `trivial`, and the results of the CI process are fine now :)